src/scenes/Wallet/components/Balances/styles.js

```diff
@@ -22,24 +22,12 @@ const styles = (theme) => ({
     fontSize: theme.sizes.font.titleMd,
     marginBottom: 2,
   },
-  table: {
-    border: theme.border,
-  },
-  tableHeader: {
-    height: theme.sizes.table.headerHeight,
-    background: theme.palette.background.grey,
-  },
   tableHeaderItemText: {
     fontWeight: theme.typography.fontWeightBold,
   },
   tableRow: {
```

Comment:
i don't think we need this anymore since it's handled in the global theme overrides

Reply:
removed and added padding to theme